### PR TITLE
*: add rev flag to get command

### PR DESCRIPTION
**Pull request by gyuho**
/cc @xiang90 @heyitsanthony 

Thanks.

### CODE REVIEW

File: etcdctl/ctlv3/command/get_command.go [hunk 1]
```diff
@@ -45,6 +46,7 @@ func NewGetCommand() *cobra.Command {
 	cmd.Flags().Int64Var(&getLimit, "limit", 0, "maximum number of results")
 	cmd.Flags().BoolVar(&getPrefix, "prefix", false, "get keys with matching prefix")
 	cmd.Flags().BoolVar(&getFromKey, "from-key", false, "get keys that are greater than or equal to the given key")
+	cmd.Flags().Int64Var(&getRev, "rev", 0, "specify the store revision")
```
Comment (xiang90) [inline etcdctl/ctlv3/command/get_command.go]: store -> kv. we also need to add this into doc.

Comment (gyuho) [inline etcdctl/ctlv3/command/get_command.go]: Will fix. Doc. You mean READMEv3.md?

Comment (xiang90) [inline etcdctl/ctlv3/command/get_command.go]: yes

**Comment by gyuho** [comment]
@xiang90 Fixed up. PTAL. Thanks!

**Comment by gyuho** [comment]
just rebased from master branch.

**Comment by xiang90** [comment]
lgtm

**Pull request merged**